### DISCUSSION TITLE
Não está referenciando o IValidatable

### DISCUSSION THREAD

**Discussion by amarinafroes** [balta-io/7196#44] [Perguntas e Respostas] [answered]
Boa Tarde.
Após seguir os passo a passo da instalação do package do flunt, ao inserir o using flunt.Validations
o mesmo não aparece a a opção de usar o IValidatable depois dos dois pontos ( : ), segue print abaixo:

![image](https://user-images.githubusercontent.com/89412365/217048029-9a49e604-8eef-41a1-88bb-fb26fec9a365.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @amarinafroes, como vai?

Acredito que você instalou a versão mais nova do Flunt que conteve mudanças na estrutura.
Recomendo instalar a mesma do curso que é a 1.0.5 (Se não me engano).

Você pode alterar a versão do Flunt no seu arquivo `.csproj` e depois executar novamente o comando `dotnet restore`.
Se você instalou o pacote em mais projetos, precisa repetir este passo acima em todos eles.

**Reply by amarinafroes** [reply]
entendi obrigada por responder consegui
